clients/ts/signalr/src/WebSocketTransport.ts

```diff
@@ -57,7 +57,11 @@ export class WebSocketTransport implements ITransport {
             };
 
             webSocket.onerror = (event: Event) => {
-                const error = (event instanceof ErrorEvent) ? event.error : null;
+                let error: any = null;
+                // ErrorEvent is a browser only type we need to check if the type exists before using it
+                if (typeof ErrorEvent !== "undefined" && event instanceof ErrorEvent) {
+                    error = event.error;
+                }
                 reject(error);
```

Comment:
This will just be `null` in Node right? That seems non-ideal. Is there no error parameter to this callback in Node?

Reply:
The value of `event` in node:
`{"AT_TARGET": 0, "BUBBLING_PHASE": 0, "CAPTURING_PHASE": 0, "NONE": 0, "bubbles": false, "cancelBubble": false,"cancelable": false, "defaultPrevented": false, "eventPhase": 0, "isTrusted": false, "returnValue": false, "scoped": false, "timeStamp": 0, "type": ""}`

Reply:
:( ok then